securedrop/journalist_app/admin.py

```diff
@@ -59,6 +59,7 @@ def update_submission_preferences():
         form = SubmissionPreferencesForm()
         if form.validate_on_submit():
             # The UI prompt ("prevent") is the opposite of the setting ("allow"):
+            flash(gettext("Preferences saved."), "submission-preferences-success")
```

Comment:
the places to update such that this line is under test are: 
* in `securedrop/tests/functional/journalist_navigation_steps.py` there is a class `JournalistNavigationStepsMixin` which contains the steps used for our selenium tests. The methods you want to look at are ` _admin_disallows_document_uploads` and `_admin_allows_document_uploads`. [Here](https://github.com/freedomofpress/securedrop/blob/de48b1b9f9a7a7426499ec7177e23ac5d7f16925/securedrop/tests/functional/journalist_navigation_steps.py#L277) is an example where we are asserting that a flashed message appears. 
* in `securedrop/tests/test_journalist.py` there are two unit tests for updating the submission preferences in `test_prevent_document_uploads` and `test_no_prevent_document_uploads`. You can add an assert here too ([example](https://github.com/freedomofpress/securedrop/blob/de48b1b9f9a7a7426499ec7177e23ac5d7f16925/securedrop/tests/test_journalist.py#L1385)).